edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,148 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions message-top-bar" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Software Installation and Setup
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You have chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
+      with installation and setup instructions. You can also <a href="{{software_download_url}}" target="_blank">open the installation window directly.</a>
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <h3>
+    {% blocktrans %}
+      Here is your unique exam code. You will be asked for it during the setup.
+    {% endblocktrans %}
+    </h3>
+    <h2> {{exam_code}}<span class='copy-to-clipboard'></span></h2>
+    <p>
+      {% blocktrans %}
+        Do not share this code. It is linked to your {{platform_name}} account and can be used only once.
```

Comment:
Replace this block with these numbered steps:

1. Open the proctoring software installation window and follow the instructions to install and set up the proctoring software.
2. Copy this unique exam code and paste it in the proctoring software window when you are asked for it.
[Exam code]
Do not share this code. It is linked to your edX account and can be used only once.
3. Keep the proctoring session window open while you are taking the exam. If you close it, the proctoring recording is stopped and you will not successfully complete the proctored exam.
4.Return to the edX courseware to start your exam.